dev_docs/older/readme.html

```diff
@@ -93,7 +93,7 @@
 	a game on it! Please refer to our complete game playing guide to get you
 	started on this subject.
 
-	Go to the <a href="./doc/game_playing_guide.html">Game Playing Guide</a>
+	Go to the <a href="../game_playing_guide/">Game Playing Guide</a>
```

Comment:
Just wondering if relative links, such as the ones on this page, should be updated to the Jekyll permalinks for consistency with how we do it on other pages (e.g. `{{ '/game_playing_guide/' | prepend: site.baseurl }}`?